wiki/development/coding-guidelines.md

```diff
@@ -80,127 +238,233 @@ class Two {foo = 2;};
 class Three {foo = 3;};
 ```
 
-### 2.1 Reasoning
+#### 5.1.1. Reasoning
 
 Putting the opening brace in it's own line wastes a lot of space, and keeping the closing brace on the same level as the keyword makes it easier to recognize what exactly the brace closes.
 
+### 5.2. Indents
+Ever new scope should be on a new ident. This will make the code easier to understand and read. Indentations consist of 4 spaces. Tabs are not allowed.
```

Comment:
Every new scope should be on a new indent. This will make the code easier to understand and read. Indentation consists of 4 spaces. Tabs are NOT allowed!